README.md

```diff
@@ -770,7 +803,11 @@ All web server requests include an API version in the url. The current version i
 
 ## POST /workflows/:version
 
-This endpoint accepts a POST request with a `multipart/form-data` encoded body.  The two elements in the body must be named `wdl` and `inputs`.  The `wdl` element contains the WDL file to run while the `inputs` contains a JSON file of the inputs to the workflow.
+This endpoint accepts a POST request with a `multipart/form-data` encoded body.  The form fields that must be included are:
+
```

Comment:
Yes, that is inconsistent, I believe the documentation is wrong but I'll investigate.